src/main/java/frc/robot/Constants.java

```diff
@@ -82,6 +87,23 @@ public static class FieldConstants {
 
 	public static class PoseEstimator {
 		public static final double BUFFER_DURATION_SECS = 1.5;
+
+		/**
+		 * Standard deviations of the pose estimate (x position in meters, y position in
+		 * meters, and heading in radians). Increase these numbers to trust your state
+		 * estimate less.
+		 */
+		public static final Matrix<N3, N1> stateStdDevs = VecBuilder.fill(0.1, 0.1, 0.1);
+
+		/**
+		 * The base standard deviation of the vision measurement, for best accuracy
+		 * calculate the standard deviation at 2 or more points and fit a line to it and
+		 * modify this using {@link SwerveDrive#addVisionMeasurement(Pose2d, double, Matrix)}
+		 * with the calculated optimal standard deviation. (Units should be meters per
+		 * pixel). By optimizing this you can get vision accurate to inches instead of feet.
+		 * Increase these numbers to trust your vision estimate less.
+		 */
+		public static final Matrix<N3, N1> baseVisionMeasurementStdDevs = VecBuilder.fill(0.1, 0.1, 0.1);
```

Comment:
Moved these std devs to Constants.java. The vision std devs were duplicated on 2 places.